packages/rlp/CHANGELOG.md

```diff
@@ -82,6 +82,14 @@ While this is convenient, it deviates from the ESM specification and forces down
 
 Along with the breaking releases we have therefore deactivated both of these options and you might therefore need to adapt some import statements accordingly. Note that you still can activate these options in your bundle and/or transpilation pipeline (but now you also have the option _not_ to, which you didn't have before).
 
+### Uint8Array / Buffer Removal
+
+This is technically not a change from the v4 version but was already introduced in v3, but since a lot of people will likely update from a v2 `RLP` version just to mention here:
```

Comment:
```suggestion
This is technically not a change from the v4 version as it was already introduced in v3, but since a lot of people will likely update from a v2 `RLP` version, we will also mention it here:
```